contracts/staking/SlashingManager.sol

```diff
@@ -73,5 +73,4 @@ contract SlashingManager is Ownable {
     }
 
     function checkpointHalt(uint256 start) public {}
```

Comment:
We can add this function when it is actually implemented. Thoughts?

Reply:
removed, FYI slashing is WIP.